.github/workflows/build_binaries.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Update Collection binaries
+
+on:
+  workflow_call:
+
+jobs:
+  build-binaries:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.19'
+      - name: Build
+        run: make build-update-collection-v3
+      - name: Store windows (arm64) binary as action artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: update-collection-v3-sumo-windows_arm64
+          path: ./src/go/cmd/update-collection-v3/update-collection-v3-sumo-windows_arm64
+          if-no-files-found: error
+      - name: Store windows (amd64) binary as action artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: update-collection-v3-sumo-windows_amd64
+          path: ./src/go/cmd/update-collection-v3/update-collection-v3-sumo-windows_amd64
+          if-no-files-found: error
+      - name: Store linux (amd64) binary as action artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: update-collection-v3-sumo-linux_amd64
+          path: ./src/go/cmd/update-collection-v3/update-collection-v3-sumo-linux_amd64
+          if-no-files-found: error
+      - name: Store linux (arm64) binary as action artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: update-collection-v3-sumo-linux_arm64
+          path: ./src/go/cmd/update-collection-v3/update-collection-v3-sumo-linux_arm64
+          if-no-files-found: error
+      - name: Store darwin (amd64) binary as action artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: update-collection-v3-sumo-darwin_amd64
+          path: ./src/go/cmd/update-collection-v3/update-collection-v3-sumo-darwin_amd64
+          if-no-files-found: error
+      - name: Store darwin (arm64) binary as action artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: update-collection-v3-sumo-darwin_arm64
+          path: ./src/go/cmd/update-collection-v3/update-collection-v3-sumo-darwin_arm64
+          if-no-files-found: error
```

Comment:
I would go with wildcard pattern: https://github.com/actions/upload-artifact#upload-using-a-wildcard-pattern

Reply:
it seems to upload one zip with all the binaries file with the wildcard pattern, do we want that?

Reply:
in that case we don't :)